site/docs/expressions/scalar_functions.md

```diff
@@ -134,7 +134,7 @@ For reference, here are are some common output type derivations and how they can
 
 | Operation                                                    | Definition                                                   |
 | ------------------------------------------------------------ | ------------------------------------------------------------ |
-| Add item to list                                             | `add(<List<T>, T>) => List<T>`                               |
+| Add item to list                                             | `add(List<T>, T) => List<T>`                               |
```

Comment:
I assume this is the correct syntax, but let me know if I'm mistaken.

Reply:
Yes, you are correct here.